src/components/color-system/examples/dark-light-mode-example.tsx

```diff
@@ -0,0 +1,94 @@
+                <div class="app-layout">
+                    <nav class="nav">
+                        <a class="nav__logo">
+                            <svg
+                                viewBox="0 0 227 220"
+                                xmlns="http://www.w3.org/2000/svg"
+                                fill-rule="evenodd"
+                                clip-rule="evenodd"
+                                stroke-linejoin="round"
+                                stroke-miterlimit="2"
+                            >
+                                <path
+                                    class="logo__shape"
+                                    d="M222.7 75.7c-15.2-52.5-58.3-71.1-82.2-74.6C120.4-2 68.2-2.3 34.7 53.3c-26.4 43.8-7.3 79.5-7.1 108.9-.2 13.4-2.2 29.5-20.8 42-18.5 12.4 6 13.4 8.5 13.5 15.1 1 39 3.2 66.4 1.9 23.7-1.1 65-.7 102-31.1 18.5-15.2 54.1-54.2 39-112.8z"
+                                    fill="#00a69c"
+                                    fill-rule="nonzero"
+                                />
+                                <path
+                                    class="logo__text"
+                                    d="M67 127.8c1.9 1.9 4.4 2.9 7.5 2.9 4.8 0 7.9-2.4 9.5-7.2l.1-.4 10.4 2.7-.1.5c-1.1 4.5-3.5 8.2-7.2 10.9-3.7 2.7-8 4.1-12.7 4.1-5.9 0-11-2-15.1-5.9-4.1-3.9-6.2-9-6.2-15.2 0-6.1 2.1-11.3 6.2-15.2 4.1-3.9 9.2-5.9 15.1-5.9 4.7 0 8.9 1.4 12.5 4 3.6 2.7 6.1 6.3 7.5 10.7l.1.5-10.9 3.1-.1-.5c-1.2-4.9-4.2-7.3-9.1-7.3-3.1 0-5.6 1-7.5 3-1.9 2-2.9 4.6-2.9 7.8.1 2.8 1 5.4 2.9 7.4zm41.6 12.1H97.3V100h11.3v6.1c3.5-4.4 8-6.6 13.4-6.6 1.3 0 2.2.1 2.9.3l.4.1-.6 10.8H122c-9 0-13.4 5.3-13.4 16.2v13zm30.6 0h-11.3V100h11.3v2.6c2.9-2.5 6.6-3.7 11.1-3.7 5.5 0 9.8 2 12.8 6.1 3.5-4 8.2-6.1 14.1-6.1 4.3 0 8 1.5 11 4.5s4.6 7 4.6 11.7v24.8h-11.2v-22.7c0-2.4-.7-4.3-2-5.8-1.4-1.4-3-2.1-5.1-2.1-2.7 0-4.7.9-6.2 2.8s-2.3 4.8-2.3 8.6v19.2h-11.3v-22.7c0-2.4-.7-4.3-2-5.8-1.4-1.4-3-2.1-5.1-2.1-2.7 0-4.7.9-6.2 2.8s-2.2 4.8-2.2 8.6v19.2zm3-59.3c.2 1.3.8 2.4 1.7 3.3 1 1 2.1 1.4 3.5 1.4 2.4 0 4.1-1 5.1-3.1l.2-.4 4.5 1-.2.5c-.8 2.1-2.1 3.7-3.7 4.8-1.7 1.1-3.6 1.6-5.9 1.6-2.8 0-5.2-1-7.2-3s-3-4.5-3-7.6c.1-3.1 1.1-5.6 3.1-7.6s4.4-3 7.3-3c2.7 0 5 1 6.9 2.9 1.9 1.9 2.9 4.4 3 7.3v1.9h-15.3zm-31.1 8.6h-5V69h5v1.4c1.4-1.3 3-1.9 5.2-1.9 1.5 0 2.1.3 3.2 1 1 .6 1.8 1.3 2.4 2.2 1.5-1.9 3.2-3.1 6.3-3.1 2 0 3.5.6 5.1 2.2 1.5 1.5 2.2 3.4 2.2 5.8v12.7h-4.9v-12c0-1.3-.3-2.3-1.1-3-.7-.7-1.3-1.1-2.4-1.1-1.2 0-1.9.4-2.9 1.5-.8 1-.9 2.6-.9 4.5v10h-5V77.3c0-1.4 0-2.3-.7-3-.7-.7-1.3-1.1-2.4-1.1-1.3 0-2 .4-2.9 1.5-.8 1-1.2 2.5-1.2 4.5v10zm-16.5 0h-5v-26h5v26zm8.2 0h-5V69h5v20.2zm47.9-15.1c-.9-.7-1.9-1.1-3.2-1.1-1.2 0-2.3.4-3.4 1.1-.9.6-1.4 1.5-1.8 2.5h9.9c-.2-1-.7-1.9-1.5-2.5zM98.2 61.6c.6-.6 1.3-.9 2.2-.9.8 0 1.6.3 2.2.9.6.6.9 1.3.9 2.1 0 .9-.3 1.6-.9 2.1-.6.6-1.3.8-2.2.8-.9 0-1.6-.3-2.2-.8-.6-.6-.9-1.3-.9-2.1s.3-1.5.9-2.1z"
+                                    fill="#fff"
+                                    fill-rule="nonzero"
```

Comment:
You're right. I just wanted to make sure that people here will notice it, the first time I showed it. It doesn't have to become this dark, specially if we use the variable. Also we should replace that ugly icon with this one (which doesn't have the word "lime" in it):

````
<svg viewBox="0 0 227 220" xmlns="http://www.w3.org/2000/svg" fill-rule="evenodd" clip-rule="evenodd" stroke-linejoin="round" stroke-miterlimit="2"><path d="M222.7 75.7c-15.2-52.5-58.3-71.1-82.2-74.6C120.4-2 68.2-2.3 34.7 53.3c-26.4 43.8-7.3 79.5-7.1 108.9-.2 13.4-2.2 29.5-20.8 42-18.5 12.4 6 13.4 8.5 13.5 15.1 1 39 3.2 66.4 1.9 23.7-1.1 65-.7 102-31.1 18.5-15.2 54.1-54.2 39-112.8z" fill="#00a69c" fill-rule="nonzero"/><path d="M67 118.9c1.9 1.9 4.4 2.9 7.5 2.9 4.8 0 7.9-2.4 9.5-7.2l.1-.4 10.4 2.7-.1.5c-1.1 4.5-3.5 8.2-7.2 10.9-3.7 2.7-8 4.1-12.7 4.1-5.9 0-11-2-15.1-5.9-4.1-3.9-6.2-9-6.2-15.2 0-6.1 2.1-11.3 6.2-15.2 4.1-3.9 9.2-5.9 15.1-5.9 4.7 0 8.9 1.4 12.5 4 3.6 2.7 6.1 6.3 7.5 10.7l.1.5-10.9 3.1-.1-.5c-1.2-4.9-4.2-7.3-9.1-7.3-3.1 0-5.6 1-7.5 3-1.9 2-2.9 4.6-2.9 7.8.1 2.8 1 5.4 2.9 7.4zm41.6 12.1H97.3V91.1h11.3v6.1c3.5-4.4 8-6.6 13.4-6.6 1.3 0 2.2.1 2.9.3l.4.1-.6 10.8H122c-9 0-13.4 5.3-13.4 16.2v13zm30.6 0h-11.3V91.1h11.3v2.6c2.9-2.5 6.6-3.7 11.1-3.7 5.5 0 9.8 2 12.8 6.1 3.5-4 8.2-6.1 14.1-6.1 4.3 0 8 1.5 11 4.5s4.6 7 4.6 11.7V131h-11.2v-22.7c0-2.4-.7-4.3-2-5.8-1.4-1.4-3-2.1-5.1-2.1-2.7 0-4.7.9-6.2 2.8s-2.3 4.8-2.3 8.6V131h-11.3v-22.7c0-2.4-.7-4.3-2-5.8-1.4-1.4-3-2.1-5.1-2.1-2.7 0-4.7.9-6.2 2.8s-2.2 4.8-2.2 8.6V131z" fill="#fff" fill-rule="nonzero"/></svg>
````

Will you or should I do it?

Reply:
I can do it now that I'm at it anyway 🙂

Reply:
⚡

Reply:
Please check that it looks like you intended, then resolve if it does 🙏

Reply:
Great! I did a small change. I used a color `var` in the SVG instead. Sharp eyes will see the difference on "CRM" in the SVG, and super humans will see that the fill color gets slightly darker too. :) Resolve if you want ;)

Reply:
From color-system.mdx in this very PR:

> Good places to use brand colors are:
> …
> - in logos or home links
> …

Shouldn't we use static colours here then? 😄

Looking at the [brand guidelines](https://www.lime-technologies.se/brand-guidelines/) there isn't a Lime CRM logo with dark text, only one with white text on teal, and one with teal text on white…

Reply:
hmmm... yes maybe. But my intention here was showing the usage of variables even in SVGs. Maybe we can just switch the icon to some generic "home" thingy? Or have other icons in the bar that switch colors in dark mode?

Reply:
Yeah, add some other icons perhaps. I think it's good to keep the logo, as long as we use it to set the example we actually want people to follow.

On a related note, I've changed this one to use the limel-example component, so that the reader can read the source code of the example. I think that's useful for this one, right? I don't think it's particularly useful for the other "examples" though, as they are not so much examples as "figures" 🙂

Reply:
**regarding the logos:** 
Just because you're also interested in these types of discussions (which is great). These days, some professional designers don't give a 💩  about a single color representing a brand via its logo. Not sure how this happened. But it's great I'd say. Why is it great? To keep it very short, colors are very subjective and people have different opinions about them. They're viewed/seen/experienced totally differently in different conditions (light conditions, [night-shift](https://support.apple.com/en-us/HT207570) color tweaks, etc..), on different screens (based on the screen quality), on different mediums (digital, print, paper's type and its coating layer, etc..), coexistence with other colors (background color affects how you see another color on top of it), color blindness, and cultural views. 

So the hole discussion with colors and logos should be separated if you ask me. A good approach can be setting a few colors, but leaving space for the logo to adjust (use any of the colors where suitable), like what [dropbox is doing](https://techcrunch.com/2017/10/03/dropbox-brand-update-streamlines-its-logo-and-takes-aim-at-creatives/). 
![image](https://user-images.githubusercontent.com/35954987/90384212-7fef1780-e081-11ea-8403-5751ce23c317.png)

A great logo should be independent. Meaning it's its form that creates the logo, not its color. Color is a very unreliable medium to represent a brand. Let's not go deep in here, but I simply wrote those lines regarding usage of brand colors, because I know there is some zealous mindset about them; also there is a brand guidelines that talks about them. I just wanna get these colors out of my way. 😄 

I'll try to add some other icons here and make the lime logo solid ;)

**seeing the source code**
Awesome idea! Yes it does help here a lot!

Reply:
Colour for logos: Totally agree! I also just pointed it out because I assume that we have brand guidelines about it. (And also because I don't think the dark text on the teal background looks very good…)

Reply:
We don't have a guideline for it :( and this logo SADLY is not even according to our guidelines. According to the so called guidelines, there must be the word "Lime" in it, which makes it super horrible. Honestly, I don't have that much respect for things that are not well thought :P 

But you have a very good point here and I'm gonna fix it ;)